script/Ciel/Day 9/09_01_CIEL09_1.txt

```diff
@@ -115,7 +115,7 @@ My throat is parched.
 // 'Washroom' isn't really common in en_US imo, it's more Canada-go. It's all bathrooms here.
 I've only just woken up,
  so my arms and legs still feel heavy,
- but I'd better hurry up and get changed then head over to the bathroom.
+ but I'd better hurry up and get changed, then head over to the bathroom.
```

Comment:
Not sure I like this one tbh

Reply:
yeah I don't like this comma either. maybe
> better hurry up and get changed and head over to the bathroom.